plugins/test/fu-plugin-test.c

```diff
@@ -15,6 +15,7 @@ struct FuPluginData {
 void
 fu_plugin_init (FuPlugin *plugin)
 {
+	fu_plugin_init_real (plugin);
```

Comment:
Can you get away with calling `fu_plugin_set_built_version(p,FWUPD_BUILT_VERSION)` in `fu_plugin_new` instead?